.buildkite/pipeline.yml

```diff
@@ -139,97 +131,61 @@ steps:
         agents:
           slurm_mem: 20GB
-
-  - group: "Sphere Examples (Gravity wave)"
-    steps:
-
-      - label: ":computer: baroclinic wave with orographic gravity wave"
-        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --orographic_gravity_wave true --job_id sphere_baroclinic_wave_ogw --t_end 1days --regression_test true"
-        artifact_paths: "sphere_baroclinic_wave_ogw"
-
-      - label: ":computer: held suarez (ρe) equilmoist hightop gw parameterization"
-        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --z_max 45e3 --z_elem 50 --dz_top 3e3 --vert_diff true --surface_scheme bulk --moist equil --forcing held_suarez --precip_model 0M --non_orographic_gravity_wave true --job_id sphere_held_suarez_rhoe_equilmoist_hightop_nogw --dt 450secs --t_end 1days --regression_test true"
-        artifact_paths: "sphere_held_suarez_rhoe_equilmoist_hightop_nogw/*"
       
   - group: "Sphere Examples (Topography)"
     steps:
 
-      - label: ":computer: held suarez (ρe_tot) topography (dcmip)"
+      - label: ":computer: held suarez (ρe) topography (dcmip)"
         command: "julia --color=yes --project=examples examples/hybrid/driver.jl --forcing held_suarez --job_id sphere_held_suarez_rhoe_topography_dcmip --t_end 6days --topography DCMIP200 --dt 200secs --regression_test false --rayleigh_sponge true --viscous_sponge true"
         artifact_paths: "sphere_held_suarez_rhoe_dcmip_topography_dcmip/*"
 
-      - label: ":computer: baroclinic wave (ρe_tot) topography (dcmip)"
+      - label: ":computer: baroclinic wave (ρe) topography (dcmip)"
         command: "julia --color=yes --project=examples examples/hybrid/driver.jl --job_id sphere_baroclinic_wave_rhoe_topography_dcmip --topography DCMIP200 --dt 200secs --regression_test false --rayleigh_sponge true --viscous_sponge true"
         artifact_paths: "sphere_baroclinic_wave_rhoe_topography_dcmip/*"
 
-      - label: ":computer: baroclinic wave (ρe_tot) topography (dcmip): no viscous sponge"
+      - label: ":computer: baroclinic wave (ρe) topography (dcmip): no viscous sponge"
         command: "julia --color=yes --project=examples examples/hybrid/driver.jl --job_id sphere_baroclinic_wave_rhoe_topography_dcmip_rs --topography DCMIP200 --dt 200secs --regression_test false --rayleigh_sponge true"
         artifact_paths: "sphere_baroclinic_wave_rhoe_topography_dcmip_rs/*"
 
-      - label: ":computer: baroclinic wave (ρe_tot) equilmoist topography (dcmip)"
+      - label: ":computer: baroclinic wave (ρe) equilmoist topography (dcmip)"
         command: "julia --color=yes --project=examples examples/hybrid/driver.jl --moist equil --precip_model 0M --job_id sphere_baroclinic_wave_rhoe_equilmoist_topography_dcmip --t_end 6days --topography DCMIP200 --dt 200secs --regression_test false --rayleigh_sponge true --viscous_sponge true"
         artifact_paths: "sphere_baroclinic_wave_rhoe_equilmoist_topography_dcmip/*"
```

Comment:
```suggestion
```
Similarly, the next item checks the feature with the held-suarez forcing applied, so in some ways is a stricter test of this configuration. In terms of more physical tests - the effort to port some of the ClimaCore tests will introduce a replacement for this in a simpler configuration.